dashboard/app/models/pd/application/email.rb

```diff
@@ -0,0 +1,22 @@
+# == Schema Information
+#
+# Table name: pd_application_emails
+#
+#  id                 :integer          not null, primary key
+#  pd_application_id  :integer          not null
+#  application_status :string(255)      not null
+#  email_type         :string(255)      not null
```

Comment:
just curious - what is `email_type` for? receipt/reminder/etc?

Reply:
Right. It's the name of the email action. These sometimes correspond with status, but other times don't for instance `principal_approval`

I'm open to ideas for a better name

Reply:
I think that makes sense, and it will be easy to tell once there are existing rows.